### PR TITLE
Hello world

### PR DESCRIPTION
**Pull request by ciraganenicole**
### Hello Microverse 
In this pull request I: 
- Setup Linters 
- Add Index.html and style.css files

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,61 @@
...
+### Setup
+  create .github/workflows folder in your first commit for the feature branch then add a copy of .github/workflows/linters.yml to the created folder
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
```
Comment (ShoiraTa) [inline README.md]: - [ ] If you do not need these sections, please delete them to keep `readme` clean 😉

**Review by ShoiraTa — CHANGES_REQUESTED** [review]
Hi @ciraganenicole ,

:thumbsup: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

There are some issues that you still need to work on to go to the next project but you are almost there!



 ## Required Changes  :recycle: 

- [ ] Pull request should be descriptive and clear. But in currently, it is  slightly confusing, because `First pull request` is not descriptive.
I suggest to edit pull request, if you click on three dots on the right side of the pull request you will get edit pull request option. Try to paste the below lines and see how it works: 
```
### Hello Microverse 
In this pull request I: 
- Setup Linters 
- Add Index.html and style.css files
```

_Check the comments under the review._ 

***
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ciraganenicole** [comment]
Hi  ***@***.****
*I'm done with required changes*

**Review by thecodechaser — APPROVED** [review]
## Hi @ciraganenicole 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**